internal/provider/api_token_resource.go

```diff
@@ -239,7 +239,7 @@ func (r *apiTokenResource) Read(ctx context.Context, req resource.ReadRequest, r
 	}
 	var token unleash.ApiTokenSchema
 	for _, t := range tokens.Tokens {
-		if t.Secret == state.Secret.ValueString() {
+		if t.Secret == state.Secret.ValueString() || t.TokenName == state.TokenName.ValueString() {
```

Comment:
Ability to import by TokenName

Reply:
Is this safe? Tokens don't have unique names

Reply:
True, and there's no other way of identifying a token than the secret... but that seems to be a bit unuseful...

Reply:
Yessir! I do agree. I think this is another one of those places where we've found a place where our API is... uh... weird